climada/engine/test/test_impact_calc.py

```diff
@@ -101,6 +101,44 @@ def test_apply_cover_to_mat(self):
             imp.todense(), np.array([[0, 0, 1], [0, 1, 0]])
             )
 
+    def test_error_handling_mismatch_haz_type(self):
+        """Test error handling in case hazard type of hazard
+        does not appear in impf_set or exposures"""
+        haz_tc = Hazard('TC')
+        exp_tc = Exposures()
+        exp_tc.gdf['impf_TC'] = None
```

Comment:
No 100% sure whether it is optimal to have a non-valid value for the impact function id. What do you think?

Reply:
True. I replaced it with an empty list ` exp_tc.gdf['impf_TC'] = []` or would you prefer something like ` exp_tc.gdf['impf_TC'] = [1]`?

Reply:
Adapted :)